### PR TITLE
Passes PKCS11 values as argument

### PR DESCRIPTION
**Pull request by kushaldas**

### CODE REVIEW

File: src/python_x509_pkcs11/pkcs11_handle.py [hunk 1]
```diff
@@ -97,6 +97,10 @@ def __init__(
         base_url: Optional[str] = None,
         http_data: Optional[Dict[str, str]] = None,
         http_headers: Optional[Dict[str, str]] = None,
+        pkcs11_mdoule: Optional[str] = None,
```
Comment (masv3971) [inline src/python_x509_pkcs11/pkcs11_handle.py:100]: fix spelling of module

File: src/python_x509_pkcs11/pkcs11_handle.py [hunk 2]
```diff
@@ -118,17 +122,28 @@ def __init__(
...
-        try:
+        # From argument
+        if pkcs11_recreate_session:
+            self.support_recreate_session = True
 
-            self._lib = lib(os.environ["PKCS11_MODULE"])
+        try:
+            # First let us get all 3 PKCS11 variables
+            if not pkcs11_mdoule:
+                pkcs11_mdoule = os.environ["PKCS11_MODULE"]
```
Comment (masv3971) [inline src/python_x509_pkcs11/pkcs11_handle.py:132]: fix spelling of module

File: src/python_x509_pkcs11/pkcs11_handle.py [hunk 3]
```diff
@@ -118,17 +122,28 @@ def __init__(
...
+            if not pkcs11_token:
+                pkcs11_token = os.environ["PKCS11_TOKEN"]
+            if not pkcs11_pin:
+                pkcs11_pin = os.environ["PKCS11_PIN"]
+
+            self._lib = lib(pkcs11_mdoule)
```
Comment (masv3971) [inline src/python_x509_pkcs11/pkcs11_handle.py:138]: fix spelling of module

File: src/python_x509_pkcs11/pkcs11_handle.py [hunk 4]
```diff
@@ -118,17 +122,28 @@ def __init__(
...
             self._lib.reinitialize()
 
             # Open the PKCS11 session
-            self._token = self._lib.get_token(token_label=os.environ["PKCS11_TOKEN"])
+            self._token = self._lib.get_token(token_label=pkcs11_token)
```
Comment (masv3971) [inline src/python_x509_pkcs11/pkcs11_handle.py:142]: This always confuses me when working with pkcs11, token is short for token-label right?

Comment (kushaldas) [reply src/python_x509_pkcs11/pkcs11_handle.py:142]: > This always confuses me when working with pkcs11, token is short for token-label right?

Yes.

**Pull request merged**